config/fieldparams/e2e.go

```diff
@@ -0,0 +1,51 @@
+//go:build e2e
```

Comment:
Why do we need now a specific e2e preset?
Why did not we need this before?

Reply:
It's because of proposer look ahead combined with the fact that e2e was always different in slots per epoch.

we now have to match fieldparams slots per epoch as well as a ssz protobuf constant